docs/UserGuide.md

```diff
@@ -11,124 +12,217 @@
+Block another time slot for an unconfirmed existing event.<br>
+Format: `block [starting time] to/ [ending time] for/ [index]`  
+
+> Deletes the task/event at the specified `index`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `block Monday 3pm to/ 5pm for/ 3`<br>	
+
+#### Find free time slot: `find time`
+Find and list free time slot in the schedule that is equal to or longer than the specified timing (in hours).<br>
+Format: `find time [number of hours] < [number of slots to find] >`  
 
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+> If there is there is a time slot longer than the required free time slot, 
+	then the free time period will be return to you
+> By default, find time will only give a single free slot when the number of slots required is not keyed in.
+
+Examples: 
+* `find time 3 `<br>	
+	You have a minimum of 3 hours free time slot between: today 5pm - 9pm. 
+* `find time 5 3 `<br>	
+	You have a minimum of 5 hours free time slot between: Monday 2pm - 9pm, Tuesday 1pm - 6pm and Saturday 9am - 5pm. 
 
 #### Exiting the program : `exit`
+#### Shortcut : `q`
 Exits the program.<br>
 Format: `exit`  
 
 #### Saving the data 
 Address book data are saved in the hard disk automatically after any command that changes the data.<br>
 There is no need to save manually.
 
+## Time Format 
+FlexiTrack support various timing input. Here are some examples! 
+
+#### Exact timing 
+User Input | Timing information read by FlexiTrack
+---------- | :---------------
+21 June 2018 4pm | Jun 21 16:00:00 
+1st January 7.20 | Jan 01 07:20
+April 22nd 5am | Apr 22 05:00
+
+#### Relative timing 
+If today is 1st of February a relative timing input is also possible with FlexiTrack
+
+User Input | Timing information read by FlexiTrack
+---------- | :---------------
+Tomorrow 4pm | Feb 02 16:00:00 
+Next week 720am | Feb 08 07:20
+3 weeks 2 pm  | Feb 22 14:00 
+next month 8am | Mar 01 08:00 
+
+#### Notes on FlexiTrack timing  
+1. FlexiTrack does support year. However, make sure that you also specify the hour of the timing 
+	as FlexiTrack will choose timing over year when it is uncertain. 
+2. When you do not specify the exact timing, FlexiTrack will assign your task to be 7.59 for due
+	date and starting time, and 16.59 for ending time. 
+
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your previous Address Book folder.
+       the file that contains the data of your previous FlexiTrack folder.
        
 ## Command Summary
 
```

Comment:
The 'Command summary' section will be used heavily by our testers. Try to make it easy to use as possible. e.g. give examples for each command.